### PR TITLE
Documentation updates re: install

### PR DESCRIPTION
**Pull request by cragun47**
Signed-off-by: Brian Cragun <bcragun@merative.com>

Small additions to documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,13 @@ cd csvtofhir
 # create virtual environment and create an "editable" install
 python3 -m venv venv && source venv/bin/activate && python3 -m pip install --upgrade pip setuptools
 python3 -m pip install -e .[dev]
+
+# From the venv: install the packages
+python setup.py install
```
Comment (dixonwhitmire) [inline README.md:21]: The `setup.py install` step is redundant as the libraries are install in the virtual environment by the previous command, `python3 -m pip install -e .[dev]`

A note we could add though is that for folks using the Z shell on OS X or Linux you will need to use double-quotes around the command:
```shell
python3 -m pip install -e ".[dev]"
```
The square brackets need to be escaped or quoted, due to the way Z shell expansion works. Here's a handy [Stack Overflow Article](https://stackoverflow.com/questions/30539798/zsh-no-matches-found-requestssecurity)

File: README.md [hunk 2]
```diff
@@ -16,6 +16,13 @@ cd csvtofhir
...
+
+# From the venv: build
+python setup.py build
```
Comment (dixonwhitmire) [inline README.md:24]: This step isn't required as part of the setup. In other words if we run through the preceding steps we will have an "editable" install in our virtual environment which allows us to execute tests or use the `csvtofhir` CLI.

If we would like to include this command, it should be in a separate section on building a distribution which in our case would be a `wheel file`.

**Review by dixonwhitmire — COMMENTED** [review]
Hi @cragun47 ! Thank you so much for taking the new project for a spin! I think you have certainly found some issues with the existing documentation. When you have a moment please review my comments and let me know what you think. I didn't want to suggest alternate workflows or verbiage too much, as I think the docs will benefit more from a set of fresh eyes.

Thanks!

File: README.md (hunk 1 above)
Comment (cragun47) [reply README.md:21]: Ahh the `python3 -m pip install -e .[dev]` failed for me.  That's why I had to figure out the individual installs.
Yes, adding the note to use quotes would be helpful.

**Comment by dixonwhitmire** [comment]
We will add build support and publishing to PyPi in #9

**Comment by dixonwhitmire** [comment]
Closing this issue, as noted above to provide build integration in #9

**Pull request closed without merging**